src/tests/strfmt.rs

```diff
@@ -62,6 +62,7 @@ fn test_values() {
     vars.insert("x".to_string(), "X".to_string());
     vars.insert("long".to_string(), too_long.clone()); // len=10
     vars.insert("hi".to_string(), "hi".to_string());
+    vars.insert("unicode".to_string(), "ಠ_ಠ".to_string());
```

Comment:
can you add a quick tests just validating that `">{unicode:^7}<"` returns the same thing for both `strfmt` and rust's standard `format!` macro?